easybuild/easyconfigs/c/ccache/ccache-3.7.11-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+easyblock = 'ConfigureMake'
+
+name = 'ccache'
+version = '3.7.11'
+
+homepage = 'https://ccache.dev/'
+description = """Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by 
+caching previous compilations and detecting when the same compilation is being done again"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```

Comment:
We're using a non-system toolchain here to keep control of the compiler that is used for building `ccache` (since relying on the system compilers leaves of "vulnerable" to using outdated/too recent compilers).

The downside of this is that this installation depends on `GCCcore`, which limits its effective use (it can only be used in conjunction with other modules using a toolchain based on the same GCC version).

Since `ccache` is a pretty straightforward tool to build and it has no other (non-build) dependencies, we can use a different approach: use `toolchain = SYSTEM` (and rename the easyconfig to `ccache-3.7.11.eb`) and add `('GCC', '9.3.0')` as a build dependency (which loads the correct `binutils`, so we can drop the separate `binutils` build dep).

That way, there will not be a runtime dependency on `GCCcore` anymore, which doesn't make sense anyway since `ccache` is statically linked.

Not using `GCC(core)` as a toolchain implies that EasyBuild won't set up the build environment (except maybe a very minimal one, see https://github.com/easybuilders/easybuild-framework/pull/3399), but that shouldn't matter at all for building `ccache`...

Reply:
done om https://github.com/Darkless012/easybuild-easyconfigs/pull/4